### ISSUE TITLE
Unable to change default Rich Text to Ckeditor on V4

### ISSUE THREAD

**Issue by mirza99** [strapi/strapi#12363]
## Bug report
Build stacks when creating Ckeditor plugin

### Describe the bug

I followed this steps : https://docs.strapi.io/developer-docs/latest/guides/registering-a-field-in-admin.html
to change de default rich text.
The command `yarn develop` stucks at 23% and the node process goes up to 130% of CPU when the plugin is added.

### Steps to reproduce the behavior

I followed this steps : https://docs.strapi.io/developer-docs/latest/guides/registering-a-field-in-admin.html

### Screenshots

![image](https://user-images.githubusercontent.com/5538015/151879436-9911b710-079a-4d1b-985c-035b11282cff.png)

### Code snippets

If applicable, add code samples to help explain your problem.

### System

- Node.js version: v14.18.0
- NPM version:
- Strapi version: 4.0.6
- Database: mysql
- Operating system: Mac OS Big Sur

**Comment by derrickmehaffy** [comment]
Can't reproduce on v4.0.6

## System

- Node.js version: v14
- NPM version: v6
- Strapi version: 4.0.6
- Database: N/A
- Operating system: Linux Mint 20.3

**Comment by dinesh4900** [comment]
When having a npm package in package.json and if I try to add similar type of package I've got this issue

**Comment by DomDew** [comment]
I had the same bug on 4.0.5, updating to 4.0.7 solved the issue.

**Comment by vinayg-cp** [comment]
Yes update to latest strapi version, you can also follow this plugin's setup instructions: 
https://github.com/Roslovets-Inc/strapi-plugin-ckeditor5

**Comment by derrickmehaffy** [comment]
> Yes update to latest strapi version, you can also follow this plugin's setup instructions: https://github.com/Roslovets-Inc/strapi-plugin-ckeditor5

Closing this issue is this is what we recommend for using CKEditor

**Comment by mirza99** [comment]
I tried to recreate the project from scratch, twice. And it doesn't worked, even with the version 4.0.7. When I started the server with `yarn develop --watch-admin` the server started but I got the error 404 (like mentioned in the tutorial) but not able to create a new admin under the port 8000 or 8080. 

I got around with @vinayg-cp suggestion, thank you ! :)

**Comment by FabioBachi** [comment]
@mirza99 Did you figure out a way to customize the CKEditor's options? Like hiding some of the features... 
I also managed to install CKEditor with @vinayg-cp suggestion!

**Comment by mirza99** [comment]
@FabioBachi  Did not try it, sorry.

**Comment by arthur-bri** [comment]
I reproduced too this issue today (the 23% one with webpack) with all Strapi version > 4.0.0.
The major drawback of the [strapi-plugin-ckeditor5](https://github.com/Roslovets-Inc/strapi-plugin-ckeditor5) plugin is the  absence of configuration for the moment on Strapi V4 (see the author comment: https://github.com/Roslovets-Inc/strapi-plugin-ckeditor5/issues/36). On my side, it is a prerequesite.

To make the tutorial works, I had to downgrade Strapi to exact version v4.0.0, but I don't really like this approach (it can bring back issues). So it's still an issue here...
